### ISSUE TITLE
Rectangular manhattan plot - color the signal with 2 different significance lines

### ISSUE THREAD

**Issue by dianacornejo** [YinLiLin/CMplot#129]
Hello, I wonder if its possible to color the signals above the 2 significance lines in the same way.
What I'm seeing is for example those below 3x10-7 they are being color correctly and those below 2.5x10-6 are shown in gray. It is possible to change this behaviour for all off them without caring if they are below -6 or -7 to have the same colors???
This is an example of my manhattan plot. Thank you

<img width="2779" alt="Screenshot 2024-08-01 at 3 33 39 PM" src="https://github.com/user-attachments/assets/9f645f05-2b24-425b-87a6-2eac9d01e15e">

**Comment by YinLiLin** [comment]
You may try to use highlight function.

**Comment by EasperOne** [comment]
Agreed, just collect the SNPs above the lower line into a factor. I think the alternate colors here is demonstrated in the README
